src/UserStory/Entries.js

```diff
@@ -69,8 +60,10 @@ class Entries extends Component {
         if (isValid) {
             console.log("data taken, reset fields")
             this.setState(initialState)
```

Comment:
The state doesn't get set to initialState when isValid is true. The forms and error messages doesnt get cleared when a form is submitted. Trying to debug now.